src/components/TabSelect/_TabButton.js

```diff
@@ -14,8 +16,11 @@ const getBgColor = (selected, secondary, theme) => {
 const Styled = component => styled(component)`
   cursor: pointer;
   margin-bottom: -1px;
-  margin-right: 5px;
-  padding: ${props => (props.small ? '5px 10px' : '10px 20px')};
+  margin-right: ${spacing('xxs')};
+  padding-top: ${props => (props.small ? spacing('xs') : spacing('xs'))};
+  padding-bottom: ${props => (props.small ? spacing('xs') : spacing('xs'))};
+  padding-left: ${props => (props.small ? spacing('small') : spacing('base'))};
+  padding-right: ${props => (props.small ? spacing('small') : spacing('base'))};
```

Comment:
Btw. the difference between small and normal tabs is almost erased with these changes, but I think that shouldn't be a big problem:

![image](https://user-images.githubusercontent.com/1216874/49220731-3e16cd00-f3d7-11e8-8196-0c594685d5d3.png)

Otherwise I'd consider `medium` instead `base` for horizonal paddings of _normal_ tab selectors..

Reply:
I also tried
```
  padding-top: ${props => (props.small ? spacing('xs') : spacing('base'))};
  padding-bottom: ${props => (props.small ? spacing('xs') : spacing('base'))};
  padding-left: ${props =>
    props.small ? spacing('small') : spacing('medium')};
  padding-right: ${props =>
    props.small ? spacing('small') : spacing('medium')};
```
It feels like there's too much empty space around the text and I don't really want to make the font larger. I don't really think there is a use case for this and I'd rather too similar than too huge.

<img width="321" alt="screenshot 2018-12-03 at 17 54 45" src="https://user-images.githubusercontent.com/92439/49388782-10a68800-f725-11e8-9491-e11f81f6654e.png">